tests/conda_package_testfiles/test_conda_bear/git/config

```diff
@@ -0,0 +1,11 @@
+[core]
+	repositoryformatversion = 0
+	filemode = true
+	bare = false
+	logallrefupdates = true
+[remote "origin"]
+	url = ../test_conda_bear_remote/
+	fetch = +refs/heads/*:refs/remotes/origin/*
+[branch "master"]
+	remote = origin
+	merge = refs/heads/master
```

Comment:
Yes. Ok since we didn't talk about this when we could, I will explain it now.

Conda packages need your code to be hosted somewhere else (usually a github repo). The tool will try and fetch the URL of your repo from the `.git/config` otherwise it will ask you for the URL where the sources are hosted.

I had to put it under `git` (instead of `.git`) because otherwise git will track that as a subrepo and things get really weird really fast. I just rename it while testing.